### PR TITLE
[Doc Issue] [Steps to set up CICD pipeline is out of date][2483120]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/use-CICD-template.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d76f772ae40a586bfce522e7fec11e3bb433aecb/msteams-platform%2Ftoolkit%2Fuse-CICD-template.md) | 95 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0e015454-cefa-4825-aebf-4145bfc2c149) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/use-CICD-template.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d767c95d479cb597835b491e3dd3303bfa7a797a/msteams-platform%2Ftoolkit%2Fuse-CICD-template.md) | 95 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7dfda628-591d-4250-ae25-361c5800245b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/use-CICD-template.md [hunk 1]
```diff
@@ -46,8 +46,17 @@ You can create the following workflow templates with GitHub:
 **Teams Toolkit in Visual Studio Code**
 
 1. Create a new Teams app project using Teams Toolkit.
-1. Select **Teams Toolkit** icon :::image type="icon" source="../assets/images/teams-toolkit-v2/add-API/api-add-icon.png" alt-text="api icon"::: from the left navigation bar.
+
+1. Select **Teams Toolkit** icon :::image type="icon" source="../assets/images/teams-toolkit-v2/add-API/api-add-icon.png"::: from the left navigation bar.
```
Comment (v-habibaumme) [inline msteams-platform/toolkit/use-CICD-template.md:50]: left pane. remove "navigation". We don't use navigation as per MSTP standards

File: msteams-platform/toolkit/use-CICD-template.md [hunk 2]
```diff
@@ -81,7 +90,7 @@ To set up pipelines with Azure DevOps for CI/CD:
 
 * Create workflow templates.
 
-  * Visual Studio Code
+  * Visual Studio Code.
```
Comment (v-habibaumme) [inline msteams-platform/toolkit/use-CICD-template.md:93]: Remove period from "Visual Studio Code"

File: msteams-platform/toolkit/use-CICD-template.md (hunk 1 above)
Comment (v-silakshmi) [reply msteams-platform/toolkit/use-CICD-template.md]: changes are done

File: msteams-platform/toolkit/use-CICD-template.md (hunk 2 above)
Comment (v-silakshmi) [reply msteams-platform/toolkit/use-CICD-template.md]: removed the period.

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**